### PR TITLE
Improve UI

### LINKED ISSUE #174: Move reflection and studio labels to a seperate section on personcard
Currently, the studio and reflection labels are stacked up together with other contact information. This makes the app very confusing to look at. Lets move the labels to a separate portion of the app so that it looks neater

### LINKED ISSUE #176: Add icons to label information.
Currently, each person's information is just stacked up on top of one another without any proper labelling. This is confusing and makes the app less user friendly. Lets add icons next to each label so that the user can easily pick out information that the user requires.

### LINKED ISSUE #177: General Improvements to UI
Currently, the text and labels in the UI for contacts are packed tightly together and are very unpleasant to look at. Lets improve the User Interface by spacing out text and increasing borders to make the UI for user friendly

### PR DESCRIPTION
**Pull request by delishad21**
Currently, the User Interface has a lot of issues that make the application very cluttered and confusing to use. Lets add multiple fixes and improves on the user experience by changing up the user interface. 

Fixes #177 
Fixes #176 
Fixes #174

### CODE REVIEW

File: src/main/resources/view/PersonListCard.fxml
Comment (Zer0Legion) [inline src/main/resources/view/PersonListCard.fxml:1]: Very subtle but useful quality of life changes! The app definitely looks much better as a whole, without the UI feeling like it was overdone.

File: src/main/resources/images/emailicon.png
Comment (jayllo-c) [inline src/main/resources/images/emailicon.png:1]: This icon adds to the aesthetic of the UI! Great job!

File: src/main/resources/view/PersonListCard.fxml [hunk 1]
```diff
@@ -2,38 +2,61 @@
...
           </minWidth>
         </Label>
-        <Label fx:id="name" text="\$first" styleClass="cell_big_label" />
+        <Label fx:id="name" styleClass="cell_big_label" text="\$first" />
       </HBox>
       <FlowPane fx:id="tags" />
-      <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
-      <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
-      <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
+         <HBox alignment="CENTER_LEFT" spacing="5">
```
Comment (jayllo-c) [inline src/main/resources/view/PersonListCard.fxml:34]: Nice job editting the fxml file to add the icons into the UI. The code is both succinct and correct in the implementation!

**Review by jayllo-c — COMMENTED** [review]
Great job editing the UI!

File: src/main/resources/images/emailicon.png
Comment (danielleloh) [inline src/main/resources/images/emailicon.png:1]: great that you added icons and made the ui prettier :) i think aesthetics are just as important :)

File: src/main/resources/view/PersonListCard.fxml
Comment (Pughal77) [reply src/main/resources/view/PersonListCard.fxml:1]: Yes this is a clean addition to UI

**Review by Pughal77 — COMMENTED** [review]
Nice implementation

**Pull request merged**